### PR TITLE
Fix spacing to core command line help docs

### PR DESCRIPTION
**Pull request by alfiedotwtf**

### CODE REVIEW

**Comment by alfiedotwtf** [comment]
When merged, could the stellar documentation online please be updated to reflect changes:

  https://www.stellar.org/developers/stellar-core/software/admin.html

File: docs/software/admin.md [hunk 1]
```diff
@@ -801,13 +799,6 @@ Stellar-core can be gracefully exited at any time by delivering `SIGINT` or
    and you may need to remove the file before it will restart. 
    Otherwise, all components are designed to recover from abrupt termination.
 
-Stellar-core can also be packaged in a container system such as Docker, so long 
-as `BUCKET_DIR_PATH`, `TMP_DIR_PATH`, and the database are stored on persistent
```
Comment (MonsieurNicolas) [inline docs/software/admin.md]: you removed too much here: the requirement for persistent volumes still exist

File: src/main/main.cpp [hunk 2]
```diff
@@ -122,14 +122,14 @@ usage(int err = 1)
           "      --base64             Use base64 for --printtxn and --signtxn\n"
           "      --catchup-at SEQ     Do a catchup at ledger SEQ, then quit\n"
           "                           Use current as SEQ to catchup to "
-          "'current'"
```
Comment (MonsieurNicolas) [inline src/main/main.cpp]: clang-format is the one that split the lines randomly like this, you can safely join this line and the next one while you're making this change

File: src/main/main.cpp [hunk 3]
```diff
@@ -122,14 +122,14 @@ usage(int err = 1)
...
+          "'current' "
           "history checkpoint\n"
           "      --catchup-complete   Do a complete catchup, then quit\n"
           "      --catchup-recent NUM Do a recent catchup for NUM ledgers, "
           "then quit\n"
           "      --catchup-to SEQ     Do a catchup to ledger SEQ, then quit\n"
           "                           Use current as SEQ to catchup to "
-          "'current'"
```
Comment (MonsieurNicolas) [inline src/main/main.cpp]: same: you can merge with the next line

File: src/main/main.cpp (hunk 2 above)
Comment (alfiedotwtf) [reply src/main/main.cpp]: Ah. Yep done.

File: src/main/main.cpp (hunk 3 above)
Comment (alfiedotwtf) [reply src/main/main.cpp]: Ditto

File: docs/software/admin.md (hunk 1 above)
Comment (MonsieurNicolas) [reply docs/software/admin.md]: you forgot to address this comment (don't remove Docker & persistent volume mention)

Comment (alfiedotwtf) [reply docs/software/admin.md]: Sorry! Weird, I didn't see this comment originally.

Comment (MonsieurNicolas) [reply docs/software/admin.md]: no problem it happens.
You missed the note following this paragraph too btw (look for TMP_DIR_PATH)

**Comment by MonsieurNicolas** [comment]
cool, thank you

**Pull request merged**